### DISCUSSION TITLE
Logitech keyboard disconnecting right after connect

### DISCUSSION THREAD

**Discussion by RodBrowning** [blueman-project/blueman#2512] [Q&A]
blueman: 2.4
BlueZ: 5.72
Distribution: amd64
Desktop environment:  2.4.2+mint1+wilma

* [x] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

I pressed and held the yellow "1" key. After the light started blinking rapidly, it connected but then disconnected suddenly.

**Comment by ThunderousLeft** [comment]
Same issue with a Microsoft Surface precision mouse

On Manjaro Linux 24.0.8 with blueman 2.4.3

**Comment by nxtkofi** [comment]
Same problem here with logitech Pop Mouse on Arch+Hyprland
